### PR TITLE
Extract shared load_llm_config utility

### PR DESCRIPTION
**Pull request by simonrosenberg**
## Summary
- Extracts the duplicated inline LLM config loading pattern (read file, validate, return `LLM`) into a shared `load_llm_config()` helper in `benchmarks/utils/llm_config.py`
- Updates all 7 `run_infer.py` files to use the shared helper, removing ~5 lines of boilerplate from each

## Test plan
- [ ] Verify imports resolve correctly in CI
- [ ] Run any benchmark inference to confirm LLM config loading still works

🤖 Generated with [Claude Code](https://claude.com/claude-code)

### CODE REVIEW

**Comment by simonrosenberg** [comment]
@openhands resolve conflicts

File: benchmarks/utils/llm_config.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+from __future__ import annotations
+
+from pathlib import Path
+
+from openhands.sdk import LLM
+
+
+def load_llm_config(config_path: str | Path) -> LLM:
```
[the remark this answers is not in the document]
Comment (simonrosenberg) [reply benchmarks/utils/llm_config.py:8]: Added comprehensive test coverage in a325dfa.

19 test cases covering:
- ✅ Valid JSON config files (minimal and full configs)
- ✅ Missing files raise `ValueError` with path in message
- ✅ Malformed JSON raises `ValidationError`
- ✅ Edge cases (empty file, permissions, unicode, extra fields)
